wallet/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation 'androidx.preference:preference:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.0-beta4'
     implementation "org.dashj:dashj-core:$dashjVersion"
-    implementation 'com.google.protobuf:protobuf-java:3.4.0'
+
```

Comment:
I would have thought we would have replaced this dependency with `protobuf-javalite`.  Where will our app get the code for this library?  Is it included in another dependency like dashj (bundled)?